idunn/places/pj_poi.py

```diff
@@ -56,7 +56,13 @@ def get_subclass_name(self):
         return self.get_class_name()
 
     def get_raw_opening_hours(self):
-        opening_hours_dict = self.get('OpeningHours', {})
+        return self.get_raw_hours_kind('OpeningHours')
+
+    def get_raw_happy_hours(self):
+        return self.get_raw_hours_kind('HappyHours')
```

Comment:
This field is not defined in the source data, we could probably ignore happy hours in this class.